apps/api/v2/package.json

```diff
@@ -30,6 +30,7 @@
     "@calcom/platform-enums": "*",
     "@calcom/platform-libraries": "npm:@calcom/platform-libraries@0.0.44",
     "@calcom/platform-libraries-0.0.2": "npm:@calcom/platform-libraries@0.0.2",
+    "@calcom/platform-libraries-1.2.3": "npm:@calcom/platform-libraries@1.2.3",
```

Comment:
Same here it's temporary, will be fixed once we release a new version of platform libraries.